src/CLR/Core/CLAUDE.md

```
@@ -488,7 +491,102 @@ they're not generics-related.
 
 ---
 
-## 13. Build + debug workflow
+## 13. Object header aliasing - monitor lock vs generic TypeSpec
+
+`CLR_RT_HeapBlock::m_data` is a union, and two of its members overlap
+exactly:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)